File: Examples/VisualBasic/Articles/HidingDisplayOfZeroValues.vb
Imports Microsoft.VisualBasic
Imports System.IO

Imports Aspose.Cells

Namespace Articles
    Public Class HidingDisplayOfZeroValues
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)
            ' Create a new Workbook object
            Dim workbook As New Workbook(dataDir & "book1.xlsx")

            ' Get First worksheet of the workbook
            Dim sheet As Worksheet = workbook.Worksheets(0)

            ' Hide the zero values in the sheet
            sheet.DisplayZeros = False

            ' Save the workbook
            workbook.Save(dataDir & "output.xlsx")
            ' ExEnd:1


        End Sub
    End Class
End Namespace

File: Examples/CSharp/Articles/HidingDisplayOfZeroValues.cs
using System;
using System.IO;

using Aspose.Cells;

namespace Aspose.Cells.Examples.CSharp.Articles
{
    public class HidingDisplayOfZeroValues
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();

            // Create a new Workbook object
            Workbook workbook = new Workbook(sourceDir + "sampleHidingDisplayOfZeroValues.xlsx");

            // Get First worksheet of the workbook
            Worksheet sheet = workbook.Worksheets[0];

            // Hide the zero values in the sheet
            sheet.DisplayZeros = false;

            // Save the workbook
            workbook.Save(outputDir + "outputHidingDisplayOfZeroValues.xlsx");

            Console.WriteLine("HidingDisplayOfZeroValues executed successfully.\r\n");
        }
    }
}
